packages/app-shortcuts/android/src/main/java/io/capawesome/capacitorjs/plugins/appshortcuts/AppShortcutsHelper.java

```diff
@@ -79,4 +89,11 @@ public static List<ShortcutInfoCompat> createShortcutInfoCompatList(JSArray shor
         }
         return shortcutInfoCompatList;
     }
+
+    @Nullable
+    public static Bitmap decodeBase64(@NonNull String base64) {
```

Comment:
```suggestion
    private static Bitmap decodeBase64(@NonNull String base64) {
```

This should not be public as it's only used by this class.